rafs/src/metadata/direct_v6.rs

```diff
@@ -61,6 +64,12 @@ use storage::device::{
 };
 use storage::utils::readahead;
 
+// Use to store chunk info pre inode, Oour build is actually single-threaded,
```

Comment:
Use to store chunk info per inode, Our build is actually single-threaded

Reply:
fixed